docs/virtual_machines/creating_vms.md

```diff
@@ -205,3 +244,33 @@ VirtualMachineCluster{Instancetype,Preference} and directly used PVC
 ```shell
 virtctl create vm --instancetype=my-instancetype --preference=my-preference --volume-pvc=my-pvc
 ```
+
```

Comment:
Can you move these examples up to the respective subheadings? (Instancetype/Preference). IMO that makes it easier to find them.

Reply:
Done